FogCreek/Glitch-Community
# Muddy moose: componentize EditCollectionColorPop

## Links
* Remix link https://muddy-moose.glitch.me
* Manuscript link https://glitch.manuscript.com/f/cases/3328626/

## GIF/Screenshots:
![Screen Shot 2019-05-23 at 2 52 19 PM](https://user-images.githubusercontent.com/938722/58278594-63730b00-7d6a-11e9-90e2-aed30c581a73.png)

## Changes:
* componentize EditCollectionColorPop
* make Color input component

## How To Test:
* log into https://muddy-moose.glitch.me
* create a collection
* clicking "Color" on the collection sidebar should open the color picker popover
* clicking Random should set the collection to a random color
* clicking the color chip should make the color adjustable via a system color picker
* editing the hex input should change the color when its a valid CSS color, and warn when it isn't
